src/validator.rs

```diff
@@ -14,6 +14,10 @@
 // You should have received a copy of the GNU General Public License
 // along with ledgeracio.  If not, see <http://www.gnu.org/licenses/>.
 
+//! Validator commands: implements operations on the validators whitelisted in
+//! the allowlist, e.g. rotating session keys, set payment target, announcing
```

Comment:
```suggestion
//! Validator commands: implements operations on the allowed validators,
//! e.g. rotating session keys, set payment target, announcing
```

Ledgeracio does not use the term “whitelist”.